### PR TITLE
Label::getStringNumLines returns 0 instead of -1 when containing an empty string

### PR DESCRIPTION
**Pull request by Pisces000221**
It used to be -1, now it's 0 when containing an empty string. It only returns -1 when string is nullptr.

### CODE REVIEW

File: cocos/2d/CCLabel.cpp [hunk 1]
```diff
@@ -772,9 +772,9 @@ int Label::getStringNumLines() const
 {
     int quantityOfLines = 1;
     
-    unsigned int stringLen = _currentUTF16String ? cc_wcslen(_currentUTF16String) : 0;
-    if (stringLen == 0)
-        return (-1);
+    unsigned int stringLen = _currentUTF16String ? cc_wcslen(_currentUTF16String) : -1;
+    if (stringLen == -1)
```
Comment (WenhaiLin) [inline cocos/2d/CCLabel.cpp:776]: The return value will be 1 of your modification when stringLen is 0.It's not expected results.

**Comment by Pisces000221** [comment]
Sorry, I made a mistake. Unsigned 0 minus 1 equals 4294967295, not -1. It should work correctly now.

**Comment by dumganhar** [comment]
@Dhilan007 
This issue was fixed in your Pull Request, right?

**Comment by WenhaiLin** [comment]
@dumganhar right.

**Pull request closed without merging**